src/Microsoft.Extensions.DataIngestion/Chunkers/DocumentTokenChunker.cs

```diff
@@ -32,17 +32,22 @@ public DocumentTokenChunker(Tokenizer tokenizer, int chunkSize, int chunkOverlap
             _chunkOverlap = chunkOverlap >= 0 ? chunkOverlap : throw new ArgumentOutOfRangeException(nameof(chunkOverlap));
         }
 
-        public Task<List<DocumentChunk>> ProcessAsync(Document document, CancellationToken cancellationToken = default)
+        public Task<List<DocumentChunk>> ProcessAsync(string text, CancellationToken cancellationToken = default)
```

Comment:
this API can be made internal and it can return just `List<DocumentChunk>` since it does not perform any async work. Or even better: to avoid the need of allocating a new list every time it's called by the other chunker, it could accept just `string text` and `List<DocumentChunk>` and just add new chunks to the list and return `void`